recipes/jemalloc/all/conanfile.py

```diff
@@ -52,8 +52,6 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
-        if self.settings.compiler.get_safe("libcxx") == "libc++":
```

Comment:
@mathbunnyru 
You're hitting the problem I was trying to avoid here.
No idea how to fix this. Maybe recent versions of clang have fixed this.

I suggest opening an issue on the jemalloc issue tracker.

Reply:
I checked - clang-8 and clang-9 doesn't work.
But clang-10 works perfectly fine.

Reply:
So since we use clang-10, I will just add check, for clang-9 and less to raise an error if compiler version is old